packages/flutter/lib/src/cupertino/list_section.dart

```diff
@@ -0,0 +1,803 @@
+///           ),
+///           CupertinoListTile(
+///             title: const Text('Push to master'),
+///             leading: Container(
+///               width: double.infinity,
+///               height: double.infinity,
+///               color: CupertinoColors.systemRed,
+///             ),
+///             additionalInfo: const Text('Not available'),
+///           ),
+///           CupertinoListTile(
+///             title: const Text('View last commit'),
+///             leading: Container(
+///               width: double.infinity,
+///               height: double.infinity,
+///               color: CupertinoColors.activeOrange,
+///             ),
+///             additionalInfo: const Text('12 days ago'),
+///             trailing: const CupertinoListTileChevron(),
+///             onTap: () => Navigator.of(context).pushNamed('lastCommit'),
+///           ),
+///         ],
+///       ),
+///     );
+///   }
+/// }
+/// ```
+/// {@end-tool}
+///
+/// {@tool snippet}
+/// Creates an "Inset Grouped" [CupertinoListSection] containing
+/// notched [CupertinoListTile]s with `leading`, `title`, `additionalInfo` and
+/// `trailing` widgets.
+///
+/// ```dart
+/// class MyStatefulWidget extends StatelessWidget {
+///   const MyStatefulWidget({Key? key}) : super(key: key);
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return CupertinoPageScaffold(
+///       child: CupertinoListSection.insetGrouped(
+///         header: const Text('My Reminders'),
+///         children: <CupertinoListTile>[
+///           CupertinoListTile.notched(
+///             title: const Text('Open pull request'),
+///             leading: Container(
+///               width: double.infinity,
+///               height: double.infinity,
+///               color: CupertinoColors.activeGreen,
+///             ),
+///             trailing: const CupertinoListTileChevron(),
+///             onTap: () => Navigator.of(context).pushNamed('openPr'),
+///           ),
+///           CupertinoListTile.notched(
+///             title: const Text('Push to master'),
+///             leading: Container(
+///               width: double.infinity,
+///               height: double.infinity,
+///               color: CupertinoColors.systemRed,
+///             ),
+///             additionalInfo: const Text('Not available'),
+///           ),
+///           CupertinoListTile.notched(
+///             title: const Text('View last commit'),
+///             leading: Container(
+///               width: double.infinity,
+///               height: double.infinity,
+///               color: CupertinoColors.activeOrange,
+///             ),
+///             additionalInfo: const Text('12 days ago'),
+///             trailing: const CupertinoListTileChevron(),
+///             onTap: () => Navigator.of(context).pushNamed('lastCommit'),
+///           ),
+///         ],
+///       ),
+///     );
+///   }
+/// }
+/// ```
```

Comment:
for samples, we should probably use the new extracted sample code setup @gspencergoog created
if you look around the code you should find other examples (e.g. AppBar, just search for "See code in examples" in the source).

Reply:
There's also a [README](https://github.com/flutter/flutter/blob/master/examples/api/README.md#authoring) for this.

Reply:
I think this [Snippets](https://github.com/flutter/flutter/blob/master/examples/api/README.md#snippets) is a better one for this case.

Reply:
This is the [commit](https://github.com/flutter/flutter/pull/78732/commits/c6c2225522f83b43dfd8da2a59aad249e9132364) I made regarding this issue.